abdulbasit-web/css/style.css

```diff
@@ -0,0 +1,169 @@
+*{
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body{
+  background: #F6F7F9;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+
+header{
+  width: 100%;
+  border-bottom: 1px solid #333;
+  box-shadow: 0 4px 2px -2px gray;
+  z-index: 1;
+}
+
+.wrrapper{
+  width: 60vw;
+  margin: 0 auto;
+}
+
+header .logo{
+  width: 70px;
+  height: auto;
+}
+
+header .logo img{
+  width: 100%;
+  
+}
+
+.wrrapper nav{
```

Comment:
We didn't actually require you to use flex, we aimed to use inline-blocks and floats but anyway it's ok as long as its working fine